crates/usvg-parser/src/font.rs

```diff
@@ -0,0 +1,109 @@
+// Copyright 2024 the SVG Types Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use crate::stream::{ByteExt, Stream};
+use crate::Error;
+
+/// Parses a list of font families and generic families from a string.
+pub fn parse_font_families(text: &str) -> Result<Vec<FontFamily>, Error> {
+    let mut s = Stream::from(text);
+    let font_families = s.parse_font_families()?;
+
+    s.skip_spaces();
+    if !s.at_end() {
+        return Err(Error::UnexpectedData(s.calc_char_pos()));
+    }
+
+    Ok(font_families)
+}
+
+/// A type of font family.
+#[derive(Clone, PartialEq, Eq, Debug, Hash)]
+pub enum FontFamily {
```

Comment:
Why do we need this if we didn't need it before?

Reply:
It's used in the parse_font_families function